ui/src/config/section/infra/primaryStorages.js

```diff
@@ -109,6 +109,16 @@ export default {
       defaultArgs: { enabled: true },
       show: (record) => { return record.state === 'Disabled' }
     },
+    {
+      api: 'updateStoragePool',
+      icon: 'control-outlined',
+      label: 'label.action.edit.nfs.options',
+      message: 'message.action.edit.nfs.options',
+      dataView: true,
+      popup: true,
+      show: (record) => { return (record.type === 'NetworkFilesystem' && record.hypervisor === 'KVM' && record.state === 'Maintenance') },
+      component: shallowRef(defineAsyncComponent(() => import('@/views/infra/NFSOptsPrimaryStorage.vue')))
+    },
```

Comment:
Probably as discussed privately, move the option add NFS mount options in the edit pool form. Show the field only for supported hypervisors/parameters. And show the remount warning when the value is changed. From what I remember, update network form kind of does the same thing for offering change

Reply:
Done